www/downloads.html

```diff
@@ -1,146 +1,171 @@
+<ul>
+<li>
+<strong>on Windows</strong>, by start a command prompt with <a
+  href="https://technet.microsoft.com/en-us/library/cc947813.aspx"> administrator privileges</a> and launch the installer (In the section "Installation Troubleshooting" there is the complete installation guide);
+</li>
+<li>
+<strong>on Linux/MacOS</strong>, by launching the installer as root with command <kbd>sudo java -jar jolie-1.8.1.jar</kbd>.
+</li>
+</ul>
+<div class="panel panel-default">
+<div class="panel-heading">
+<strong>Installation Troubleshooting</strong>
+</div>
+<div class="panel-body">
+<div class="accordion" id="accordion2">
+<div class="accordion-group">
+<div class="accordion-heading">
+<a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion2" href="#collapseOne"> <strong>I get “command not found” after I installed Jolie under MacOs X El Capitan (10.10.11).</strong> </a>
+<hr>
+</div>
+<div id="collapseOne" class="accordion-body collapse">
+<div class="accordion-inner">
+
+In MacOs X El Capitan (10.10.11) it is not possible to install Jolie using the default values provided by the installer. When prompted by the installer, insert e.g., <code>/usr/local/lib/jolie</code> as the directory of installation of Jolie and <code>/usr/local/bin</code> as the directory of the launchers.
+<hr>
+</div>
+</div>
+</div>
+<div class="accordion-group">
+<div class="accordion-heading">
+<p><a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion2" href="#collapseTwo"> <strong>I get “Error: Could not find or load main class jolie.Jolie” after I installed Jolie under Linux/MacOs X. </strong> </a></p>
+</div>
+<div id="collapseTwo" class="accordion-body collapse">
+<div class="accordion-inner">
+<p>The launchers deployed by the installer use the environment variable <code>JOLIE_HOME</code> to set the classpath and launch Jolie. As reported by the installer at the end of the installation, it is possible to set <code>JOLIE_HOME</code> with the command <code>echo ‘export JOLIE_HOME=“/usr/lib/jolie”’ &gt;&gt; ~/.bash_profile</code></p>
+<p>However, some versions of Linux/MacOs X do not <a
+href="http://ss64.com/bash/source.html">source</a> the <code>.bash_profile</code> file. If, when trying to run Jolie you get the message <code>“Error: Could not find or load main class jolie.Jolie”</code>, it could be the case your shell is not sourcing file <code>.bash_profile</code>. To fix it, edit your <code>.bashrc</code> in you home directory by appending at its end the line <code>source .bash_profile</code>.</p>
+</div>
+</div>
+</div>
+<div class="accordion-group">
+<div class="accordion-heading">
+<a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion2" href="#collapseOne"> <strong>Complete guide for installing on Windows.</strong> </a>
+<hr>
+</div>
+<div id="collapseOne" class="accordion-body collapse">
+<div class="accordion-inner">
+<p>1) Write <code>C:\Jolie</code> and do Enter.</p>

```

Comment:
These two lines seem repeated. Is it a typo?
I suggest being a bit more specific, what does "do Enter" mean. Press the Enter key?

Reply:
No no, it's not a mistake.
The first is for the folder in which to install Jolie.
The second is for the folder in which to install the Jolie variables.
Because I saw that many people on the second step click only "Enter" but doing so after the variables are not installed and after Jolie does not go.